frontend/README.md

```
@@ -0,0 +1,96 @@
+```bash
+npm run dev      # 開発サーバーを起動
+npm run build    # 本番用ビルド
+npm run lint     # ESLint を実行
+npm run test     # テストをウォッチモードで実行
+npm run test:run # テストを1回実行
+npm run preview  # 本番ビルドをプレビュー
+```
+
+## プロジェクト構成
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)